src/Api/index.ts

```diff
@@ -8,10 +8,12 @@ export type WhpptApi = { app: AppApi; site: SiteApi; page: PageApi, security: Se
 export type WhpptApiConstructor = () => WhpptApi;
 
 const http = Http(process.env.NEXT_PUBLIC_BASE_API_URL);
+const storageHttp = StorageHttp(process.env.NEXT_PUBLIC_STORE_API);
```

Comment:
Remove anything storage related since were going to move the saveFile onto the http. Also the NEXT_PUBLIC_STORE_API wouldnt have been needed since the api url was always the same as the http.